### PR TITLE
basic and bearer auth working

### PR DESCRIPTION
**Pull request by fadnesscharlie**
## Functionality of Auth working
- [x] Bearer Auth
- [x] Basic Auth
- [x] Add/Post user
- [x] Get User
## TODO in another PR
- Update User
- Delete User

### CODE REVIEW

**Comment by MuckT** [comment]
@fadnesscharlie added a sample test for our error handler along with fixing some lint errors.

File: package.json [hunk 1]
```diff
@@ -24,6 +24,7 @@
   "homepage": "https://github.com/follow-the-nft/follow-the-nft-api#readme",
   "dependencies": {
     "axios": "^0.24.0",
+    "base-64": "^1.0.0",
```
Comment (MuckT) [inline package.json:27]: Remove if not going to use.

File: src/middleware/basic.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
 'use strict';
 
-const base64 = require('base64-js');
+// const base64 = require('base64-js');
```
Comment (MuckT) [inline src/middleware/basic.js:3]: Remove unused dependency.

File: src/models/collections.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+class Collection {
```
Comment (MuckT) [inline src/models/collections.js]: Lets remove this.

**Review by jeremywbrazell — APPROVED** [review]

**Review by MuckT — APPROVED** [review]
LGTM!

**Pull request merged**